docs/howto/partitions.rst

```diff
@@ -48,14 +48,20 @@ To add partition support to an application, two basic changes are needed:
 #. Enable the feature
 
    Use the :class:`Features <craft_parts.Features>` class to specify that the
-   application will use partitions:
+   application will use partitions.  This should be called from an overridden
+   method in your Application subclass:
 
    .. code-block:: python
 
      from craft_parts import Features
 
-     Features.reset()
-     Features(enable_partitions=True)
+     class SnackcraftApplication(Application):
+
+         ...
+
+         @override
+         def _enable_craft_parts_features(self) -> None:
+             Features(enable_partitions=True)
 
    .. NOTE::
```

Comment:
Since this clears up an understandable point of confusion, we can promote this to a regular paragraph. Also, as a general principle I use callouts judiciously because they fight for the reader's attention.

The text could also be refined:

```rest
    You can only be enable partitions with the :class:`Features
    <craft_parts.Features>` class from craft-parts. In craft-applications
    there's a similarly-named :class:`AppFeatures
    <craft_application.AppFeatures>` class which serves a similar purpose, but
    it can't enable partitions.
```